### PR TITLE
feat:RDCC-1589 Assign access- add organisationIdenitifier in get user response

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
JIRA link (if applicable)

https://tools.hmcts.net/jira/browse/RDCC-1589

Change description

feat:RDCC-1589 Assign access- add organisationIdenitifier in get user response

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[X] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java [hunk 1]
```diff
@@ -10,6 +11,9 @@
 
 @NoArgsConstructor
 public class ProfessionalUsersEntityResponse {
+
+    @JsonProperty
+    private String organisationIdentifier;
```
Comment (KOTLAPRA23) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java:16]: do we required organisationIdentifier in both classes? (super and sub classes)

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java [hunk 2]
```diff
@@ -21,4 +25,12 @@
     public void setUserProfiles(List<ProfessionalUsersResponse> userProfiles) {
         this.userProfiles = userProfiles;
     }
+
+    public void setOrganisationIdentifier(String organisationIdentifier) {
+        this.organisationIdentifier = organisationIdentifier;
+    }
+
+    public String getOrganisationIdentifier() {
```
Comment (KOTLAPRA23) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java:33]: getter is required here?

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponseWithoutRoles.java [hunk 3]
```diff
@@ -11,6 +12,9 @@
 @NoArgsConstructor
 public class ProfessionalUsersEntityResponseWithoutRoles {
 
+    @JsonProperty
+    private String organisationIdentifier;
```
Comment (KOTLAPRA23) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponseWithoutRoles.java:16]: do we required organisationIdentifier in both classes? (super and sub classes)

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java (hunk 1 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java:16]: There is no parent-child relationship here

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponseWithoutRoles.java (hunk 3 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponseWithoutRoles.java:16]: There is no parent-child relationship here

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java (hunk 2 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java:33]: yes required for object to Json parsing.

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponseWithoutRoles.java (hunk 3 above)
Comment (KOTLAPRA23) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponseWithoutRoles.java:16]: ok. i saw some code before thats why asked..

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java (hunk 1 above)
Comment (KOTLAPRA23) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java:16]: ok.

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java (hunk 2 above)
Comment (KOTLAPRA23) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java:33]: ok

**Pull request merged**